src/harness/testcases/WINNF_FT_S_GRA_testcase.py

```diff
@@ -58,6 +58,85 @@ def setUp(self):
   def tearDown(self):
     pass
 
+  @winnforum_testcase
+  def test_WINNF_FT_S_GRA_1(self):
+    """Federal Incumbent present in the PAL frequency range requested by the CBSD
+     who is inside the DPA Neighborhood.
+     
+     grant responseCode = 0 and in heartbeat responseCode = 501(SUSPENDED_GRANT)
+     or  grant responseCode = 400
+    """
+    # Trigger SAS to load DPAs
+    self._sas_admin.TriggerLoadDpas()
+    # Trigger SAS to de-active all the DPAs 
+    self._sas_admin.TriggerDpaActivation({'activate':False})
```

Comment:
This should be `TriggerBulkDpaActivation`, right?

Reply:
yes,  you are right.
done, thank you.

Reply:
yes, you are right.
done, thank you.